docs/TSUNAMI_METADATA_CONVENTIONS.md

```diff
@@ -0,0 +1,31 @@
+# Tsunami Metadata Conventions
+
+In this document we describe the metadata conventions used for GeoNet sensors that monitor tsunami.
+
+## Overview
+
+GeoNet has two types of tsunami monitoring stations: tide gauges and DART buoys. The distinction is made by the primary purpose of the station; whether it exists principally to record tsunami arrivals on-shore (via a tide gauge) or in the deep ocean (via a DART buoy).
+
+The management of DART data and metadata is an evolving space, and until this is more established we refrain from documenting it here.
+
+Tide gauge stations operated by GeoNet are not measuring absolute sea level, as they are not fully calibrated to the mean sea level. Data obtained by GeoNet tide gauges are measuring water height above the sensor.
+
+Tide gauge data and metadata mostly follow the conventions of seismic data and metadata, and unless stated otherwise in this document these conventions are what apply to tide gauge data and metadata. The sibling document `SEISMIC_METADATA_CONVENTIONS.md` details these conventions. Here we make reference only to the differences between the tide gauge metadata conventions and those of seismic metadata.
+
+## Location Code
+
+For tide gauges, the location code indicates primary and backup sensors at the station. For example, a station may have sites at location codes `41` and `42`, and these location code are used to distinguish the respective data streams from the sensors at each site. In this way, we can think of the location code for tide gauges as a "sensor code". 
+
+## Naming Conventions When Moving or Installating Different Sensors
+
+Tide gauges are sensitive to the overlying water column, which is principally influenced by sensor depth and local tides. To correctly represent data measured at tsunami monitoring sites, tidal constituents must be derived.
+
+Where in seismic data small changes to sensor position can have substantial impacts on the data, the same is not true for tide gauges. 
+
+If a station position changes such that tidal constituents are still the same, the station code will not change.
```

Comment:
I think the sensor/station distinction is quite important here. It is right to say what you have here, but it doesn't capture the more inclusive case of when it is just one sensor that moves, in which case a new station may be opened but the old station may not be decomissioned (i.e. if other sensors are still there).

Reply:
Feel free to change it back. My main point here was to change the sentence from the "negative" form to a form that is valid for tide gauges, and mostly captures use cases there..

Reply:
Done